### DISCUSSION TITLE
I get the terminal piece to work but when I install the Target on Xcode I always get the error "periphery command not found"

### DISCUSSION THREAD

**Discussion by artud2000** [peripheryapp/periphery#644] [General]
I was able to successfully run periphery on the terminal, however when I create the new Target to run in Xcode and I add the buildPhase Script I get the following error when I try to build **"periphery command not found"**

**Comment by ccalderon9** [comment]
<img width="533" alt="Screenshot 2023-09-03 at 9 07 14 PM" src="https://github.com/peripheryapp/periphery/assets/29626980/c0c370c5-c3f4-4e5a-88f5-b751188ffcab">

Check 'For install builds only'! That did the trick for me.

**Reply by levous** [reply]
that just skips the script unless installing. Doesn't fix the issue.

**Comment by agwolff** [comment]
I have the same issue and checking "for install builds only" nothing happens, it doesn't even run the script

**Comment by ghost711** [comment]
Try using the full path to periphery in the script. For me that's ```/opt/homebrew/bin/periphery```

**Comment by mcstoufer** [comment]
```bash
export PATH="$PATH:/opt/homebrew/bin"
PERIPHERY=$(mint which periphery | tail -1) # YMMV for whatever provides the periphery binary.
$PERIPHERY scan ...
```

**Comment by mmalesevic** [comment]
In XCode 26+ you musn't forget to set `User Script Sandboxing = NO` in Build Settings...
